db/migrate/20160719142419_create_projects.rb

```diff
@@ -0,0 +1,11 @@
+class CreateProjects < ActiveRecord::Migration[5.0]
+  def change
+    create_table :projects do |t|
+      t.string :title
```

Comment:
if you have validation inside model you should reflect in on DB too. At least present